x/blob/keeper/keeper.go

```diff
@@ -37,28 +34,16 @@ func NewKeeper(
 	}
 }
 
-func (k Keeper) Logger(ctx sdk.Context) log.Logger {
-	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
-}
-
 // PayForBlobs consumes gas based on the blob sizes in the MsgPayForBlobs.
 func (k Keeper) PayForBlobs(goCtx context.Context, msg *types.MsgPayForBlobs) (*types.MsgPayForBlobsResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
-
-	// GasPerBlobByte is a versioned param from version 3 onwards.
-	var gasToConsume uint64
-	if ctx.BlockHeader().Version.App <= v2.Version {
-		gasToConsume = types.GasToConsume(msg.BlobSizes, k.GasPerBlobByte(ctx))
-	} else {
-		gasToConsume = types.GasToConsume(msg.BlobSizes, appconsts.GasPerBlobByte(ctx.BlockHeader().Version.App))
-	}
```

Comment:
this was intensionally removed, right? 

seems like its causing failing tests in gas_test.go, previously the code path only followed with params lookup, I'll just remove the 1060 gas for params lookup for now

Reply:
Yes, it is on purpose.